### DISCUSSION TITLE
大佬，可以在别人撤回的时候加一个撤回的提示么

### DISCUSSION THREAD

**Discussion by shnke666666** [huiyadanli/RevokeMsgPatcher#650] [Ideas]
大佬们，防撤回也用了很久，可以提出一点小小的建议么，就是在别人撤回之后给出一个提示，不然也不知道别人是撤回了还是没有撤回，就像微信原本的撤回之后也是有提示的，要是能完善这个的话就很棒了，谢谢大佬们！

**Comment by bananaguys** [comment]
对，上次我引用了别人撤回的内容，尴尬死了
